notes.md

```diff
@@ -0,0 +1,17 @@
+# Making a dag out of deps
+
+## Runtime DAG
+
+Which mods can I split up into separate @ folders?
```

Comment:
Please don't write this, it's most annoying when you play on a joint op and the organizing community decided to rename or change folders around for no reason! 

Rather write it in the sense of checking which PBOs rely on which and what can be removed (especially in terms of modular mods).

Reply:
It's actually for our own modpack :) I want to make it more modular so I can debug stuff. It would be fun to do a `hemtt bisect` someday...

My current plan is just to output a DOT file. People can then do with that what they wish.